eZ/Publish/Core/Persistence/Legacy/Content/Gateway/ExceptionConversion.php

```diff
@@ -886,4 +886,27 @@ public function insertRelation( RelationCreateStruct $struct )
             throw new RuntimeException( 'Database error', 0, $e );
         }
     }
+
+    /**
+     * Returns all Content IDs for a given $contentTypeId.
+     *
+     * @param int $contentTypeId
+     *
+     * @return int[]
+     */
+    public function getContentIdsByType( $contentTypeId )
```

Comment:
Confussion avoided in https://github.com/ezsystems/ezpublish-kernel/commit/1b9e7396b0cb35c7169f2b9030351a2ef84562e0 ;)